testdata/scripts/first.txt

```diff
@@ -0,0 +1,10 @@
+exec devnet &
```

Comment:
just like dealbot, add this to TestMain

Reply:
Right, then we only have to assume that `lotus` is available in the PATH, compiled for 2k sectors.

Reply:
:+1: you could also do something like `[!exec:lotus] skip 'test needs lotus to run'` if you would rather have the tests be skipped rather than fail when lotus isn't installed.